src/actions/ads.jsx

```diff
@@ -57,7 +23,8 @@ const fetchAdsFailure = error => ({
   payload: error,
 });
 
-const fetchAds = (regionid, localityid, rooms, page) => (
+const fetchAds = (regionid, localityid, rooms, iscombinedbathroom, hasbalcony, minPrice, maxPrice,
```

Comment:
you could pass just an object like `options` and pass it directly to `params`

that's pretty hard to remember more than ~3 function parameters, try to avoid it